src/task6/task6

```diff
@@ -0,0 +1,22 @@
+function reverseArray(array) {
+    let reversedArray = [];
+    for (let i = 0; i < array.length; i++) {
+        reversedArray[i] = array[array.length - i - 1];
+    }
+    return reversedArray;
+}
+
+function reverseArrayInPlace(array) {
+    for (let i = 0; i <= Math.floor((array.length) / 2); i++) {
+        array.splice(i, 0, array[array.length - 1])
+        array.pop();
```

Comment:
One quick improvement you can do:
What does **.pop** method do and return?
How can you use it?

Reply:
You are the best student I've had so far
Well done!